DetDescr/src/DetDescr/HcalGeometry.cxx

```diff
@@ -19,49 +19,81 @@ HcalGeometry::HcalGeometry(const framework::config::Parameters& ps)
   ecal_dx_ = ps.getParameter<double>("ecal_dx");
   ecal_dy_ = ps.getParameter<double>("ecal_dy");
   verbose_ = ps.getParameter<int>("verbose");
-  horizontal_parity_ = ps.getParameter<int>("horizontal_parity");
```

Comment:
let's change the "horizontal_parity" parameter to "back_horizontal_parity" too?

Reply:
Done!